keras_eval/metrics.py

```diff
@@ -113,15 +118,20 @@ def metrics_top_k(y_probs, y_true, concepts, top_k, round_decimals=7):
     accuracy = utils.round_list(average_accuracy_k, round_decimals)
 
     metrics['average'] = OrderedDict([('accuracy', accuracy if len(accuracy) > 1 else accuracy[0]),
-                                      ('precision', round(utils.safe_divide(sum(average_precision), total_samples),
+                                      ('sensitivity', round(utils.safe_divide(sum(average_sensitivity), len(concepts)),
```

Comment:
`average_sensitivity` is the sensitivity computed for each class right? Not the averaged sensitivity.

Would it be clearer to call it something like `sensitivity_per_class`? I think this can be simplified to something like:

`mean_sensitivity = np.mean(sensitivity_per_class)`

Reply:
I changed the nomenclature, however we were not doing `np.mean` because of the NaN values.